sdk/ai/Azure.AI.Projects/src/Custom/Inference/Inference.cs

```diff
@@ -0,0 +1,275 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ClientModel;
+using System.ClientModel.Primitives;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Azure.AI.Inference;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.Identity;
+using Azure.AI.OpenAI;
+using OpenAI.Chat;
+
+namespace Azure.AI.Projects
+{
+    // Data plane generated client.
+    /// <summary> The AzureAI service client. </summary>
+    public partial class AIProjectClient : ClientConnectionProvider
+    {
+        #nullable enable
+        public ChatCompletionsClient GetChatCompletionsClient()
+        {
+            ChatCompletionsClientKey chatCompletionsClientKey = new();
+            ChatCompletionsClient chatClient = this.Subclients.GetClient(chatCompletionsClientKey, () => CreateChatCompletionsClient());
+            return chatClient;
+        }
+
+        private ChatCompletionsClient CreateChatCompletionsClient()
+        {
+            ClientConnection connection = this.GetConnection(typeof(ChatCompletionsClient).FullName!);
+
+            if (!connection.TryGetLocatorAsUri(out Uri? uri) || uri is null)
+            {
+                throw new InvalidOperationException("Invalid URI.");
+            }
+            uri = new Uri($"https://{uri.Host}/models");
+
+            AzureAIInferenceClientOptions options = new AzureAIInferenceClientOptions();
+            // OverrideApiVersionPolicy overrideApiVersionPolicy = new OverrideApiVersionPolicy(AIProjectClientOptions.ServiceVersion.V2025_05_15_Preview.ToString());
```

Comment:
If you're making other changes, probably remove the commented lines in this file. If not doing any other changes then it's fine.